diff --git a/node_modules/wavesurfer.js/dist/plugins/envelope.d.ts b/node_modules/wavesurfer.js/dist/plugins/envelope.d.ts
index 9e76185..429790c 100644
--- a/node_modules/wavesurfer.js/dist/plugins/envelope.d.ts
+++ b/node_modules/wavesurfer.js/dist/plugins/envelope.d.ts
@@ -32,7 +32,7 @@ export type EnvelopePluginEvents = BasePluginEvents & {
 };
 declare class EnvelopePlugin extends BasePlugin<EnvelopePluginEvents, EnvelopePluginOptions> {
     protected options: Options;
-    private polyline;
+    public polyline;
     private points;
     private throttleTimeout;
     private volume;
